docs/index.md

```diff
@@ -29,34 +29,35 @@ settings in this case.
 Write this to a file (`config.yml`):
 
 ```yaml
-cache_dir: "/tmp/symbolicator"
-bind: "0.0.0.0:3021"
+cache_dir: '/tmp/symbolicator'
```

Comment:
Happy to revert these style changes but we force autoFormat on our repo for VSCode and this is how Prettier wants our md files to look like :D

Reply:
Most changes are fine, but please revert the indentation level. Unfortunately, `mkdocs` is kinda inflexible and doesn't allow to change the indentation level (afaik), and this would break rendering of the docs. I'm also fine if you prefer to swap the markdown renderer :)

Reply:
Yeah, prettier is quite weird when it comes to indentation. Will revert.